lib/ronin/support/binary/ctypes/os/android.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+#
+# Copyright (c) 2006-2023 Hal Brodigan (postmodern.mod3 at gmail.com)
+#
+# ronin-support is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# ronin-support is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with ronin-support.  If not, see <https://www.gnu.org/licenses/>.
+#
+
+require 'ronin/support/binary/ctypes/os/linux'
+
+module Ronin
+  module Support
+    module Binary
+      module CTypes
+        class OS
+          #
+          # Contains additional types available on Android.
+          #
+          # @api semipublic
+          #
+          # @since 1.1.0
+          #
+          class Android < Linux; end
```

Comment:
One minor style issue, I prefer to put the `end` on the next line.

Reply:
Great, next time, I will follow that approach.